converters/toZigbee.js

```diff
@@ -3357,24 +3357,23 @@ const converters = {
             // Protocol description
             // https://github.com/Koenkk/zigbee-herdsman-converters/issues/1159#issuecomment-614659802
 
+            const invert = meta.mapped.meta && meta.mapped.meta.coverInverted ? !meta.options.invert_cover : meta.options.invert_cover;
+
             if (key === 'position') {
                 if (value >= 0 && value <= 100) {
-                    const invert = !(meta.mapped.meta && meta.mapped.meta.coverInverted ?
-                        !meta.options.invert_cover : meta.options.invert_cover);
                     value = invert ? 100 - value : value;
                     await tuya.sendDataPointValue(entity, tuya.dataPoints.coverPosition, value);
                 } else {
                     throw new Error('TuYa_cover_control: Curtain motor position is out of range');
                 }
             } else if (key === 'state') {
-                const isRoller = meta.mapped.model === 'TS0601_roller_blind';
```

Comment:
Inverting should only invert the position, not the state. open should still be open, not closed. The only thing we want to reverse is position.

By default it should be open = 100, close = 0

Reply:
@Koenkk That's what I thought, but the previous version sent 0 for close and 2 for open only for roller blinds. All curtain motors had 2 for close and 0 for open. My curtain motor also needs 0 for close and 2 for open though.

So there are 3 options I can think of:
* Add the previous check if model is TS0601_roller_blind but also check if zigbeeModel contains my specific model
* Apply the roller blind behaviour to all curtains
* Split up my curtain (owvfni3) from the other curtain motors [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L1380)

I'm not sure which ones need 2 for close and 0 for open or if its not necessary at all but I assume it was added for a reason.

Reply:
@DomiStyle i propose to merge `TS0601_curtain` and `TS0601_roller_blind` and check the `manufacturerName` if 2 or 0 needs to be send for open/close.

Reply:
@Koenkk Where would be the best place to store which manufacturerNames need to be inverted by default and which need to reverse open/close commands? In an array somewhere or hardcoded in the tuya_cover_control function?

Reply:
There's this issue which also seems to have trouble with the cover state: https://github.com/Koenkk/zigbee2mqtt/issues/5596

Not even stop seems to be mapped correctly in this case. Maybe we should have multiple presets and add the covers one by one once its known which direction they go?

Reply:
> edit: In the tuya.js maybe?

Yes, add a function: `isTuyaCoverInverted`, and by default return false.